### PR TITLE
Make npm scripts work on Windows too.

### PR DESCRIPTION
**Pull request by fatfisz**
Now they should work everywhere :smile:

### CODE REVIEW

**Comment by hzoo** [comment]
Thanks

File: package.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
     "bootstrap": "git submodule update --init && cd eslint && npm install",
     "eslint": "cd eslint && mocha -c tests/lib/rules/*.js -r ../eslint-tester.js",
     "test": "mocha",
-    "lint": "./node_modules/eslint/bin/eslint.js index.js babylon-to-espree test",
-    "fix": "./node_modules/eslint/bin/eslint.js index.js babylon-to-espree test --fix",
+    "lint": "node node_modules/eslint/bin/eslint.js index.js babylon-to-espree test",
+    "fix": "node node_modules/eslint/bin/eslint.js index.js babylon-to-espree test --fix",
```
Comment (loganfsmyth) [inline package.json]: Shouldn't these just rely on npm to set the path and be `eslint index.js babylon-to-espree test`?

Comment (fatfisz) [inline package.json]: That would be even better, but then how did the current form come about? 🤔

Comment (hzoo) [inline package.json]: Good point - feel free to send another PR

**Pull request merged**